backend/setup-without-docker.md

```diff
@@ -6,6 +6,8 @@ Use this guide if you do NOT want to use Docker in your project.
 
 Create and activate a virtual environment, and then install the requirements.
 
+
```

Comment:
We probbaly show include how to setup the venv or resources that may help.

Reply:
cool let me add documentation for that